src/ADAL.PCL/Flows/AcquireTokenInteractiveHandler.cs

```diff
@@ -44,11 +44,13 @@ internal class AcquireTokenInteractiveHandler : AcquireTokenHandlerBase
 
         private readonly string extraQueryParameters;
 
+        private readonly string claims;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private field 'claims'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS1144)